src/apps/loan-list/materials/stackable-material/stackable-material.tsx

```diff
@@ -63,9 +69,7 @@ const StackableMaterial: FC<StackableMaterialProps & MaterialProps> = ({
         >
           <AdditionalMaterialsButton
             showOn="desktop"
-            openDueDateModal={() =>
-              openDueDateModal && dueDate && openDueDateModal(dueDate)
-            }
+            openDueDateModal={() => handleOpenDueDateModal()}
```

Comment:
Change to `openDueDateModal={handleOpenDueDateModal}` here and below

Reply:
What difference does it make @JacobArrow ?